client/public/src/components/app.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from 'react';
+import ReactDOM from 'react-dom';
+import axios from 'axios';
+
+import Details from './details.jsx';
+import Options from './options.jsx';
+
+const App = () => {
+  const [productId, setProductId] = useState(1);
+  const [product, setProduct] = useState({});
+  const [storeId, setStoreId] = useState(1);
+  const [stores, setStore] = useState([]);
+  const [stock, setStock] = useState([]);
+  const [colors, setColors] = useState([]);
+  const [sizes, setSizes] = useState([]);
+
+  // get the products general details such as price, name and reviews
+  const getProduct = async (productId) => {
+    const response = await axios.get(`http://localhost:3000/products/${productId}`);
```

Comment:
if you have this running through your server, I think you can trim it down to just `/products/${productId}`

Reply:
oh that's right!